docs/HOWTO.md

```diff
@@ -807,19 +807,18 @@ fi
 # POSIX way to get script's dir: https://stackoverflow.com/a/29834779/12156188
 script_dir="$(cd -P -- "$(dirname -- "$(command -v -- "$0")")" && pwd -P)"
 # exec: replace current process with chezmoi init
-exec $chezmoi init --apply --clone=false --source $script_dir 
+exec $chezmoi init --apply --source=$script_dir
```

Comment:
I believe you should wrap between quotes to prevent a case where the directory have spaces.

```suggestion
exec "$chezmoi" init --apply "--source=$script_dir"
```

Or:

```suggestion
exec "$chezmoi" init --apply --source "$script_dir"
```

Reply:
Thanks, I've updated the commit with your first suggestion (accepting the suggestion in GitHub would create a new commit, which is not necessary).

Reply:
I see. I think this suggestion mechanism is only really useful when you merge commits using squash (which I also do not prefer).